pydantic/json.py

```diff
@@ -18,25 +18,27 @@ def isoformat(o: Union[datetime.date, datetime.time]) -> str:
 
 
 ENCODERS_BY_TYPE: Dict[Type[Any], Callable[[Any], Any]] = {
+    bytes: lambda o: o.decode(),
```

Comment:
I'm not clear why we're bothering to re-order this dict?

Reply:
It's a dict, so order doesn't matter, especially since we're `KeyError`ing, and sorted items make it easier to add more later. I sorted it when I moved `Path` and `Enum`.